src/heitech.zer0mqXt.core/Transport/Message.cs

```diff
@@ -73,10 +73,12 @@ internal static XtResult<TMessage> ParseRqRepMessage<TMessage>(this NetMQMessage
             bool isSuccess = Convert.ToBoolean(configuration.Serializer.Deserialize<string>(successFrame));
             if (!isSuccess)
             {
-                var exceptionText = configuration.Serializer.Deserialize<string>(payloadFrame);
-                Exception ex = configuration.DeveloperMode 
-                               ? new ZeroMqXtSocketException("Server failed with" + exceptionText)
-                               : ZeroMqXtSocketException.ResponseFailed<TMessage>();
+                Exception ex =  ZeroMqXtSocketException.ResponseFailed<TMessage>();
```

Comment:
A piece of me hates to complain here yet again - but the other piece of me just has to:
Now we unconditionally allocate a `ZeroMqXtSocketException` through `ResponseFailed`, which is then discarded in `DeveloperMode`.
I guess this one is less "severe", because it would "hit" only the `DeveloperMode`, but still 😉.

Reply:
Alright the first one was valid from where I am coming from, this one is absolutely fine for me. So I leave it at that as it has a neglible performance impact this way.

Reply:
Fair enough.